FhirToDataLake/docs/Process FHIR extensions.md

```diff
@@ -0,0 +1,101 @@
+# Process FHIR extensions  
+
+ ![image](./assets/ProcessExtensions.png)
+
+## Generate parquet data for FHIR extensions
+
+### 1.	Prepare the customized schema templates
+For each resource types, e.g. Patient, Observation, you need to prepare a **liquid template** and a related **JSON schema** file.
+- The liquid template will be used to convert the raw FHIR JSON data into target structure.
```

Comment:
@QuanWanxx Shall we mention the liquid template here is using the same syntax from FHIR-Converter repo?

Reply:
Good idea, I just found this [Filters-and-Tags](https://github.com/microsoft/FHIR-Converter/blob/main/docs/Filters-and-Tags.md), maybe we can add the reference for that?